sdk/storage/azdatalake/internal/generated/autorest.md

```diff
@@ -298,3 +298,17 @@ directive:
         replace(/err = unpopulate\((.*), "ContentLength", &p\.ContentLength\)/g, 'var rawVal string\nerr = unpopulate(val, "ContentLength", &rawVal)\nintVal, _ := strconv.ParseInt(rawVal, 10, 64)\np.ContentLength = &intVal').
         replace(/err = unpopulate\((.*), "IsDirectory", &p\.IsDirectory\)/g, 'var rawVal string\nerr = unpopulate(val, "IsDirectory", &rawVal)\nboolVal, _ := strconv.ParseBool(rawVal)\np.IsDirectory = &boolVal');
 ```
+
+### Updating service version to 2023-08-03
+```yaml
+directive:
+- from: 
+  - zz_service_client.go
+  - zz_filesystem_client.go 
+  - zz_path_client.go
+  where: $
+  transform: >-
+    return $.
+      replaceAll(`[]string{"2023-05-03"}`, `[]string{ServiceVersion}`).
```

Comment:
Why we need transformation for the service version here?

Reply:
This Paginated delete feature is applicable for service version `2023-08-03` (STG90) and later. The swagger, though has not changed in STG89 and 90, so, the service version in latest swagger is `2023-05-03` (STG89).
To support STG90 features, this transformation is needed.